module-3/backend/week_one.md

```diff
@@ -4,8 +4,20 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What is `json` and why is it important?
-2. What's the difference between `joins` and `includes` in ActiveRecord?
-3. What's an API?
-4. How do we test an internal API (in general)?
-5. What are two different ways to customize your `json`?
+1. What is `json` and why is it important? 
+     - json - javascript object notation -  is the tool used to handle javasrcipt querys, and it is important because it takes     
+    the response of a query we asked for and returns a format that can be used in a different language like ruby for example.
+2. What's the difference between `joins` and `includes` in ActiveRecord? 
+    - joins brings in another table and sort of pastes it to the starting table along a column that they have in common. 
+    Includes is a method that shows a given collumn with the table you queryed for
```

Comment:
Includes works very similar to joins, but performs some eager loading to avoid multiple queries to the database where possible.